openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -1095,6 +1105,7 @@ def process(self, instance):
                 "FTRACK_SERVER": os.environ.get("FTRACK_SERVER"),
             }
 
+        deadline_publish_job_id = None
```

Comment:
Nice catch. However, looking at the other submission type "muster" I'm pretty sure this should **always** be True for a plugin in Deadline module hehe, and the muster stuff is waaaaaaaaay outdated.